content/zh/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,175 @@
+
+团队所做的关于 Envoy 接口定义更改如下：
+
+1. [Listener addresses](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto.html?highlight=additional_addresses)
+1. [bind config](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig).
+
+对于 Istio 双栈特性支持的实现，这些修改是很重要的，它确保我们能够在 Envoy 的下游和上游连接上得到适当的支持。
+
+团队总共向 Envoy 提交了十多个 PR，其中有多半数的 PR 的目的是使 Envoy 采用双栈时对 Istio 来说更加容易。
+
+同时，在 Istio 方面，也可以在 [Issue #40394](https://github.com/istio/istio/issues/40394) 中跟踪进度。因为我们在与 Envoy 社区解决各种双栈支持遇到的问题，所以 Istio 社区方面的进展有所放缓。尽管如此，我们很高兴的宣布 Istio 1.17 中实现了对双栈特性的实验性支持！
+
+## 使用双栈的快速实验
+
+1. Enable dual stack experimental support on Istio 1.17.0+ with the following
+
+{{< text bash >}}
+$ istioctl install -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_AGENT_DUAL_STACK: "true"
+  values:
+    pilot:
+      env:
+        ISTIO_DUAL_STACK: "true"
+EOF
+{{< /text >}}
+
+1. 创建3个命名空间:
+
+    * `dual-stack`: `tcp-echo` will listen on both an IPv4 and IPv6 address.
+    * `ipv4`: `tcp-echo` will listen on only an IPv4 address.
+    * `ipv6`: `tcp-echo` will listen on only an IPv6 address.
+
+{{< text bash >}}
+$ kubectl create namespace dual-stack
+$ kubectl create namespace ipv4
+$ kubectl create namespace ipv6
+{{< /text >}}
+
+1. 在所有这些命名空间以及默认命名空间上启用 sidecar 注入：
+
+{{< text bash >}}
+$ kubectl label --overwrite namespace default istio-injection=enabled
+$ kubectl label --overwrite namespace dual-stack istio-injection=enabled
+$ kubectl label --overwrite namespace ipv4 istio-injection=enabled
+$ kubectl label --overwrite namespace ipv6 istio-injection=enabled
+{{< /text >}}
+
+1. 在命名空间中创建 `tcp-echo` 部署：
+
+{{< text bash >}}
+$ kubectl apply --namespace dual-stack -f {{< github_file >}}/samples/tcp-echo/tcp-echo-dual-stack.yaml
+$ kubectl apply --namespace ipv4 -f {{< github_file >}}/samples/tcp-echo/tcp-echo-ipv4.yaml
+$ kubectl apply --namespace ipv6 -f {{< github_file >}}/samples/tcp-echo/tcp-echo-ipv6.yaml
+{{< /text >}}
+
+1. 在默认命名空间中创建 `sleep` 部署：
+
+{{< text bash >}}
+$ kubectl apply -f {{< github_file >}}/master/samples/sleep/sleep.yaml
+{{< /text >}}
+
+1. 流量校验:
+
+{{< text bash >}}
+$ kubectl exec -it "$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')" -- sh -c "echo dualstack | nc tcp-echo 9000"
+hello dualstack
+$ kubectl exec -it "$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')" -- sh -c "echo ipv4 | nc tcp-echo.ipv4 9000"
+hello ipv4
+$ kubectl exec -it "$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')" -- sh -c "echo ipv6 | nc tcp-echo.ipv6 9000"
+hello ipv6
+{{< /text >}}

```

Comment:
The indentations on line 36-98 are not correct now. Ident 3 spaces please.

Reply:
Hi @windsonsea  done

Reply:
but I encountered below lint error when I Ident 3 spaces between line# 36-98

Reply:
`Text blocks need to not be indented, or indented by a multiple of 4 spaces`
So I think it may be unnecessary to indent here.